src/device/usbd.c

```diff
@@ -209,6 +229,8 @@ static usbd_class_driver_t const _usbd_driver[] =
       .class_code       = TUD_RNDIS_ITF_CLASS,
       .subclass         = TUD_RNDIS_ITF_SUBCLASS,
       .protocol         = TUD_RNDIS_ITF_PROTOCOL,
+      .all_subclass     = 0,
+      .all_protocol     = 0,
```

Comment:
Coding style question: Add these lines, or just leave them out (the compiler will default to setting them to 0)?

Reply:
better to explicitly assign them, else I think we can got into C++ compiler issue with non-trivial initialization error.